.github/workflows/ci.yml

```diff
@@ -214,12 +214,14 @@ jobs:
       - name: Build package
         run: yarn build:release
       - name: Publish canary version
-        run: yarn changeset publish --tag canary
+        run: yarn workspaces foreach --no-private --from '@storefront-ui/*' npm publish --access public --tag canary
```

Comment:
Not sure if `--access public` part is required -> we have this specified in `package.json` of every subpackage

Reply:
I think it should work, lets check https://github.com/vuestorefront/storefront-ui/pull/2853/commits/31a95cbb04df2413dddf28c2305c44362ecae700